src/main/java/org/apache/commons/validator/routines/checkdigit/TidDECheckDigit.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.validator.routines.checkdigit;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * German Steuer-Identifikationsnummer (TIN – in short: IdNr.) is available since 2008,
+ * has a length of 11 digits and it applies to individual persons.
+ *
+ * <p>
+ * See <a href="https://download.elster.de/download/schnittstellen/Pruefung_der_Steuer_und_Steueridentifikatsnummer.pdf">Prüfung der Steuer- und Steueridentifikationsnummer (de)</a>
+ * for more details.
+ * </p>
+ *
+ * @since 1.10.0
+ */
+public final class TidDECheckDigit extends Modulus11TenCheckDigit {
+
+    private static final long serialVersionUID = 4222306963463322195L;
+    private static final Log LOG = LogFactory.getLog(TidDECheckDigit.class);
+
+    /** Singleton Check Digit instance */
+    private static final TidDECheckDigit INSTANCE = new TidDECheckDigit();
+
+    /**
+     * Gets the singleton instance of this validator.
+     * @return A singleton instance of the class.
+     */
+    public static CheckDigit getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Constructs a modulus Check Digit routine.
+     */
+    private TidDECheckDigit() {
+        super();
+    }
+
+    /**
+     * {@inheritDoc} <br>
```

Comment:
Don't leave HTML tags open. You have paragraphs below, this break is likely not needed.